package messages;

message HandSpeeds
{
   optional float left_speed = 1;
   optional float right_speed = 2;
}